src/mqtt/qmqtt_routesubscription.h

```diff
@@ -61,6 +65,7 @@ private slots:
     explicit RouteSubscription(Router *parent = 0);
     void setRoute(const QString &route);
 
+    QPointer<Client> _client;
```

Comment:
You have chosen `QPointer` since it zeros the pointer when the `Client QObject` disappears, is this correct? Just wondering since we haven't used it yet.

Reply:
Yep, there is no guarantee that _client pointer is still valid when RouteSubscription is deleted and I wanted to be safe